pkg/app/e2e/build_test.go

```diff
@@ -12,14 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package app
+package e2e
```

Comment:
I was expecting to move pkg/app to e2e/, actually. Or maybe test/e2e.

Reply:
e2e seems better after I look throught some projects.